dashboard/client/src/pages/node/hook/useNodeList.ts

```diff
@@ -50,11 +50,31 @@ export const useNodeList = () => {
     };
   }, [getList]);
 
+  const finalSortFunc = (a: NodeDetail, b: NodeDetail) => {
+    const sortFuncs: ((a: NodeDetail, b: NodeDetail) => number)[] = [
+      // user override first
+      sorterFunc,
+      // Head node is always first
+      (a, b) => (a.raylet.isHeadNode ? 0 : 1) - (b.raylet.isHeadNode ? 0 : 1),
+      // Then sort by state
+      (a, b) => (a.raylet.state > b.raylet.state ? 1 : -1),
```

Comment:
is alive higher than dead in this case? (is it because alive starts from 'a'?)

Reply:
yeah